src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue

```diff
@@ -101,21 +106,42 @@ const defaultTemplateState = reactive({
     boardSets: computed<DashboardTemplateBoardSet[]>(() => Object.values(DASHBOARD_TEMPLATES).map((d: DashboardConfig) => ({
         ...d,
         // below values are used only for render
-        leftIcon: d.description?.preview_image ?? '',
-        // song-lang
-        iconButtonSets: [{ iconName: 'ic_external-link', tooltipText: 'Preview', eventAction: () => {} }],
+        leftIcon: d.description?.icon ?? '',
+        iconButtonSets: [{
+            iconName: 'ic_external-link',
+            // song-lang
+            tooltipText: 'Preview',
+            eventAction: () => {
+                const href = `/images/dashboard-previews/dashboard-img_${d.description?.preview_image}--thumbnail.png`;
+                window.open(href, '_blank');
+            },
+        }],
     })).slice(10 * (defaultTemplateState.thisPage - 1), 10 * defaultTemplateState.thisPage - 1)),
 });
 const existingTemplateState = reactive({
-    dashboards: computed<DashboardConfig[]>(() => [...store.state.dashboard.domainItems, ...store.state.dashboard.projectItems].filter((d) => d.name.includes(existingTemplateState.searchValue))),
+    dashboards: computed<DashboardModel[]>(() => [...store.state.dashboard.domainItems, ...store.state.dashboard.projectItems].filter((d) => d.name.includes(existingTemplateState.searchValue))),
     thisPage: 1,
     allPage: computed<number>(() => Math.ceil(existingTemplateState.dashboards.length / 10) || 1),
-    boardSets: computed<DashboardTemplateBoardSet[]>(() => existingTemplateState.dashboards.map((d: DashboardConfig) => ({
+    boardSets: computed<DashboardTemplateBoardSet[]>(() => existingTemplateState.dashboards.map((d: DomainDashboardModel & ProjectDashboardModel) => ({
```

Comment:
suggestion: There is DashboardModel type that is already declared.

```suggestion
    boardSets: computed<DashboardTemplateBoardSet[]>(() => existingTemplateState.dashboards.map((d: DashboardModel) => ({
```

Reply:
`DashboardModel` is `DomainDashboardModel | ProjectDashboardModel`.
But in this case, I need `DomainDashboardModel & ProjectDashboardModel`. 
Because, the `existingTemplateState.dashboards` is consisted by both of them